spec/javascript/components/molecules/FormGrid.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import FormGrid from 'components/molecules/FormGrid'
+
+describe('FormGrid.Item', () => {
+  test('should render FormGrid.Item', () => {
+    const wrapper = shallow(<FormGrid.Item>ABC</FormGrid.Item>)
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should render small FormGrid.Item', () => {
+    const wrapper = shallow(<FormGrid.Item small>ABC</FormGrid.Item>)
+    expect(wrapper).toMatchSnapshot()
```

Comment:
nit: I think snapshot tests should be separate from non-snapshot tests. I think it's okay to just leave this snapshot assertion out, since line 14 is a good enough test as-is

Reply:
You could add a separate test checking that it defaults to *-6 though